lib/Froxlor/Cron/Dns/PowerDNS.php

```diff
@@ -111,9 +113,15 @@ private function clearZoneTables($domains = null)
 
 	private function insertZone($domainname, $serial = 0)
 	{
-		$ins_stmt = \Froxlor\Dns\PowerDNS::getDB()->prepare("
+		if (Settings::Get('system.powerdns_mode') === 'Master') {
```

Comment:
Why not just making `type` also a variable for the prepared statement?

```php
$ins_stmt = \Froxlor\Dns\PowerDNS::getDB()->prepare("
    INSERT INTO domains set `name` = :domainname, `notified_serial` = :serial, `type` = :type
");
$ins_stmt->execute(array(
	'domainname' => $domainname,
	'serial' => $serial,
        'type' => strtoupper(Settings::Get('system.powerdns_mode'))
));
```

Reply:
Thanks for the hint here too! Its fixed also in: https://github.com/Froxlor/Froxlor/pull/974/commits/29586bd29c2ff4e9b01ac34e57143a353e9aceb3